pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -60,6 +60,9 @@ func (c *Cluster) getOSDLabels(osd OSDInfo, failureDomainValue string, portable
 	labels[portableKey] = strconv.FormatBool(portable)
 	labels[deviceClass] = osd.DeviceClass
 	labels[osdStore] = osd.Store
+	if osd.DeviceType != "" {
+		labels["device-type"] = osd.DeviceType
```

Comment:
Can we use the const here?
```suggestion
		labels[deviceType] = osd.DeviceType
```